app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :title, :body, :user_id, presence: true
```

Comment:
* [ ] You shouldn't validate the presence of a Foreign key. In usual cases, associations will do that for you :gear: